elf-edit-core-dump/extract-core-dump/Main.hs

```diff
@@ -1,21 +1,22 @@
 {-# LANGUAGE GADTs #-}
 {-# LANGUAGE RankNTypes #-}
 {-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE ImportQualifiedPost #-}
```

Comment:
Minor nit pick: we adhere to the convention of alphabetizing the `LANGUAGE` pragmas in each module, so please ensure that this is alphabetized:

```suggestion
{-# LANGUAGE GADTs #-}
{-# LANGUAGE ImportQualifiedPost #-}
{-# LANGUAGE RankNTypes #-}
{-# LANGUAGE ScopedTypeVariables #-}
```

Similar comments apply to the other uses of `{-# LANGUAGE ImportQualifiedPost #-}` in this package.